.gitignore

```diff
@@ -6,4 +6,6 @@ dist
 docker-compose.prod.yml
 .ignored_files
 .vscode
-.DS_Store
\ No newline at end of file
+.DS_Store
+.dist
+*.json
```

Comment:
where is `.dist` and `.json` coming from? I doubt that we want that to exclude all json files, and there should not be a `.dist` folder..